Automattic/jetpack
# Migrating Jetpack VideoPress class to the package

#### Changes proposed in this Pull Request:

* Migrates functionality from the VideoPress module main class to the videopress package. These are things that will be needed by the videorpess stand alone plugin

#### Other information:


#### Jetpack product discussion


pe4Cmx-40-p2

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Basically we need to test for regressions. Nothing should change using the Jetpack plugin
* Start with a connected site with only the Jetpack plugin. Connect and activate VideoPress
* Make sure you are still able to upload videos to VideoPress
* Make sure you are still able to use the old `video` block to upload a video, and confirm that the `...jetpack_vendor/automattic/jetpack-videopress/src/js/videopress-token-bridge.js?ver=0.1.x-dev` JS is loaded in the Gutenberg editor
* Go to Media, click on a VideoPress video and confirm the "Video URL" field shows a URL pointing to `https://videos.files.wordpress.com/...`
* Make sure you can delete a video and no errors are triggered
* Load your site front-end in a post where there's a video and make sure it plays normally
* Confirm that the `...jetpack_vendor/automattic/jetpack-videopress/src/js/videopress-token-bridge.js?ver=0.1.x-dev` JS is loaded
* Check the code and verify all deprecations are right

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202561463041787
